cloud_formation/lambda/downsample/dlq.py

```diff
@@ -0,0 +1,32 @@
+# Copyright 2018 The Johns Hopkins University Applied Physics Laboratory
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import boto3
+import json
+
```

Comment:
Can you add a brief comment that states the lambda's purpose? Looks like it populates the SQS dead letter queue when a lambda fails and generates an SNS msg?

Reply:
The lambda is invoked by the downsample volume SNS DLQ and it will then enqueue the message onto a downsample specific SQS queue (created by the resolution hierarchy activity)